apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -218,4 +241,8 @@ defmodule AdminAppWeb.ProductView do
   defp generate_taxon_text([head | tail], acc) do
     generate_taxon_text(tail, acc <> "#{head.name} > ")
   end
+
+  def get_total_stock(variant) do
```

Comment:
@SagarKarwande are you sure about placing this method in the `product_view`? can you justify the responsibility of the file and the this method?